tests/helpers/setup-pretender.ts

```diff
@@ -0,0 +1,18 @@
+import { TestContext } from 'ember-test-helpers';
+import Pretender from 'pretender';
+
+export default function setupPretender(hooks: NestedHooks) {
+  hooks.beforeEach(function(this: TestContext) {
+    this.server = new Pretender();
+  });
+
+  hooks.afterEach(function(this: TestContext) {
+    this.server.shutdown();
+  });
+}
+
+declare module 'ember-test-helpers' {
+  interface TestContext {
+    server: Pretender;
+  }
+}
```

Comment:
This bit here informs the type system that the QUnit test context now has a `server` property as well. All of the types for Pretender are made available automatically to the test this way!